src/components/TagSet/tagSetChangelogData.ts

```diff
@@ -9,6 +9,16 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Documentation", "Styles"],
+    notes: [
+      "Updated the styles for the UI colors, text treatment and `Clear all` button.",
```

Comment:
Just personal but how do you feel about using the oxford comma here?

Reply:
I am recent convert to using the Oxford comma.  I am trying to use it more and more, but I missed this one. :)  Admittedly, however, it does still bother me in certain situations.